app/lib/registration/sign_up_screen.dart

```diff
@@ -33,69 +38,90 @@ class SignUpScreen extends HookWidget {
       resizeToAvoidBottomInset: true,
       appBar: AppBar(
         automaticallyImplyLeading: false,
-        leading: const AppBarBackButton(),
-        title: Text(loc.signUpScreen_header),
+        leading: AppBarBackButton(
+          backgroundColor: colors.backgroundElevated.primary,
+        ),
+        title: Text(
+          loc.signUpScreen_header,
+          style: const TextStyle(fontWeight: FontWeight.bold),
+        ),
         toolbarHeight: isPointer() ? 100 : null,
+        backgroundColor: backgroundColor,
       ),
+      backgroundColor: backgroundColor,
       body: SafeArea(
         minimum: EdgeInsets.only(
           bottom: isKeyboardShown ? Spacings.s : Spacings.l + Spacings.xxs,
         ),
-        child: Column(
-          children: [
-            Expanded(
-              child: LayoutBuilder(
-                builder: (context, constraints) {
-                  return SingleChildScrollView(
-                    keyboardDismissBehavior:
-                        ScrollViewKeyboardDismissBehavior.onDrag,
-                    padding: const EdgeInsets.only(
-                      left: Spacings.s,
-                      right: Spacings.s,
-                      bottom: Spacings.xl,
-                    ),
-                    child: ConstrainedBox(
-                      constraints: BoxConstraints(
-                        minHeight: constraints.maxHeight,
-                      ),
-                      child: Align(
-                        alignment: Alignment.topCenter,
-                        child: _Form(
-                          formKey: formKey,
-                          showErrors: showErrors.value,
+        child: Container(
+          color: colors.backgroundBase.secondary,
```

Comment:
Most likely the background color should be applied everywhere, also on the "unsafe" area. So, `SafeArea` must be wrapped by container, and not vise versa.